activities/ChatPrototype.activity/js/activity.js

```diff
@@ -124,6 +144,18 @@ define(["sugar-web/activity/activity","webL10n","sugar-web/graphics/palette","su
 				return false;
             }
         };
+
+        imageUpload.onchange = function(){
+        	var image = imageUpload.files[0];
+			var reader = new FileReader();
```

Comment:
For compatibility, images should be taken only from the Journal. See how to include the Journal chooser dialog box here: https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step7.md